advanced/index.php

```diff
@@ -232,10 +238,10 @@ function queryAds($serverName) {
   <?=$viewPort ?>
   <meta name="robots" content="noindex,nofollow"/>
   <meta http-equiv="x-dns-prefetch-control" content="off">
-  <link rel="shortcut icon" href="//pi.hole/admin/img/favicon.png" type="image/x-icon"/>
-  <link rel="stylesheet" href="//pi.hole/pihole/blockingpage.css" type="text/css"/>
+  <link rel="shortcut icon" href="admin/img/favicon.png" type="image/x-icon"/>
```

Comment:
Does this need to be relative? Because there might be more instances of relative to server root URLs

Reply:
Yes, to support port and host name changes. I replaced all occurrences of this in this file.

Reply:
I'm talking about the missing slash at the beginning. Because there is at least one more case that's using `/admin` and not `admin` in this page.

https://github.com/pi-hole/pi-hole/blob/94ec8fdfcade9e7c63b816bdbdc4809c54487dda/advanced/index.php#L335

So, I'm not sure if the missing slash is intentional or not, nor can I confirm this. Just noticed the discrepancy.

Reply:
I think it would be best to have the URL be relative, since you may want URL rewriting.

For example: I want to put the pihole in a docker behind a reverse proxy (and use something like traefik to route via URLs plus HTTPS middleware).  If the URL is relative, I can have it go to https://homelab/pihole/admin/img/favicon.png and things work fine.  If it's /admin, then that won't work.

Where else are you seeing /admin?

Reply:
In the line I showed you above and here https://github.com/pi-hole/pi-hole/blob/94ec8fdfcade9e7c63b816bdbdc4809c54487dda/advanced/index.php#L63

Reply:
> I'm talking about the missing slash at the beginning. Because there is at least one more case that's using `/admin` and not `admin` in this page.

Ah, sorry, misunderstood what you meant.

In the case where it's accessible directly under the root (e.g, `https://pihole.domain.tld`) either of the two works. But there should probably be a consistent use of one of the two.

For the case that @amunchet mentions only the relative URL will work. But in that case I am not sure if the redirect will still work: 

https://github.com/pi-hole/pi-hole/blob/94ec8fdfcade9e7c63b816bdbdc4809c54487dda/advanced/index.php#L56

Reply:
Actually, I think it's fine to only support direct access instead of URL rewriting - i.e. going to http://pi.hole/admin instead of http://homeserver/pihole/admin.  I think the header rewriting won't work with relative paths, so that pretty much settles it.  I don't think there's any harm in using relative paths in the HTML parts though.

Reply:
So, why not use in the 2 places too? Just trying to understand why we are not consistent here.

Reply:
I can agree with that.  Up to @mschoettle, as this is his PR.  I don't think it will make any difference functionality-wise, but the consistency will probably "smell" better.

Reply:
I totally agree. I changed it in the 2 places to use relative URLs as well for consistency.